.github/workflows/lint.yml

```diff
@@ -52,10 +52,10 @@ jobs:
           # Don't include files like LICENSE and css files
           CHANGED_JS_FILES=$(echo "$CHANGED_FILES" | sed -e '/^.\+\(mjs\|tsx\|jsx\|js\|ts\)$/!d')
 
-          # Replace \n characters with a space
-          CHANGED_FILES_SEPARATED_BY_SPACES=$(echo "$CHANGED_JS_FILES" | sed ':a;N;$!ba;s/\n/ /g')
+          # Don't include :parts files
+          CHANGED_JS_FILES=$(echo "$CHANGED_JS_FILES" | sed -e '/^.\+\:.*$/d')
```

Comment:
Maybe just drop this since it's a bit coincidental that it's needed right now and it may be hard for a future maintainer to assess whether it can be safely removed?

Reply:
Yes that makes sense! I have removed it :smile: